mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -1907,4 +1907,17 @@
     <!-- Content description for privacy content close button -->
     <string name="privacy_content_close_button_content_description">Close</string>
 
+    <!-- Pocket recommended stories -->
+    <!-- Header text for a section on the home screen. -->
+    <string name="pocket_stories_header">Thought provoking stories</string>
+    <!-- Header text for a section on the home screen. -->
+    <string name="pocket_stories_categories_header">Stories by topic</string>
+    <!-- Text of a button allowing users to access an external url for more Pocket recommendations. -->
+    <string name="pocket_stories_placeholder_text">Discover more</string>
+    <!-- Title of an app feature. Smaller than a heading.-->
+    <string name="pocket_stories_feature_title">Powered by Pocket</string>
+    <!-- Caption which concatenates the text of "pocket_stories_feature_learn_more"  -->
+    <string name="pocket_stories_feature_caption">Part of the Firefox family. %s</string>
```

Comment:
A few things:
* We don't concatenate strings, since many languages form their sentences differently than in English. Please have only one string with one string ID
* Please always explain what the placeholder will be replaced with.
CC @Mugurell 
* We don't refer to string IDs in the comment: in fact, localizers don't necessarily see them given how our tool works